x-pack/plugins/security_solution/cypress/tasks/alerts.ts

```diff
@@ -108,8 +112,13 @@ export const closeAlerts = () => {
 };
 
 export const expandFirstAlertActions = () => {
+  // Adding wait time because.. something is loading in the background
+  // which is preventing the popup to open upon button click below.
+  // Currently not sure what
+  waitForAlerts();
+  cy.wait(500);
```

Comment:
Hardcoded waits are not allowed, please find another solution or ask for support on this.

Reply:
🔨 Fixed. Please check the amendments here

Reply:
Lots of thanks for removing all the waits ❤️